### ISSUE TITLE
i18n not working for splash and onboarding

### ISSUE THREAD

**Issue by luclu** [ethereum/mist#698]
Those screens fall back to english.

**Comment by alexvandesande** [comment]
Yes, I'm having the same issue. Haven't been able to identify why.

**Comment by ChihChengLiang** [comment]
https://github.com/ethereum/mist/blob/develop/interface/client/appStart.js#L3-L5 Remove these codes fix the issue.

**Comment by frozeman** [comment]
Im not 100% sure anymore why i put it there, but i think it is to prevent the main appstart, e.g. loading tabs etc in the splash screens and popup windows.. e.g. -> saving resources.

As the popup windows use the same metro app as the main mist interface, so it doesn't need to start all the other stuff, this is also a security protection, though this can be solved better by adding a global var in the preloaded file from the mist interface: https://github.com/ethereum/mist/blob/develop/modules/preloader/mistUI.js
